recipes/adc-streaming/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "adc-streaming" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "c1ab8aee52684ff1f1c5ae469a760a435d6c45836ead4e9a97423a4893cb92d6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/adc-streaming-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python
```

Comment:
Please add a minimum version: `python >=3.6` (this is the minimum version officially supported by conda-forge).